A74-xds-config-tears.md

```diff
@@ -0,0 +1,346 @@
+A74: xDS Config Tears
+----
+* Author(s): @markdroth
+* Approver: @ejona86
```

Comment:
:cry:

Reply:
Whoops -- sorry, not sure why you weren't listed here.  Fixed.